setup.cfg

```diff
@@ -14,7 +14,7 @@ multi_line_output=3
 force_grid_wrap=0
 combine_as_imports=True
 lines_after_imports=2
-known_first_party=pyk4a,k4a_module
+known_first_party=pyk4a,k4a_module,helpers
```

Comment:
Since helpers imports opencv with `import cv2`, does this change add an opencv dependency required during install?

Reply:
No. It just affects on sorting import smatetments in `make fmt`. Install is not required.